src/client/pages/Offer/ProductSelector/Selector.tsx

```diff
@@ -115,13 +148,19 @@ export const ProductSelector = ({
               image={image}
               checked={selectedProductsByCategory.additionalProducts[id]}
               onClick={() => {
```

Comment:
nit: this looks like code duplication - maybe consider extracting to a function but could be done later also